rune/libenclave/internal/runtime/pal/pal_linux.go

```diff
@@ -58,3 +62,83 @@ func (pal *enclaveRuntimePal) Destroy() error {
 	api := &enclaveRuntimePalApiV1{}
 	return api.destroy()
 }
+
+func (pal *enclaveRuntimePal) GetLocalReport(targetInfo []byte) ([]byte, error) {
+	if pal.version >= 3 {
```

Comment:
I think you are unknown to what pal.version means here. pal.version indicates which pal api version is implemented by an enclave runtime PAL. So do you mean the older enclave runtime PAL implement get local report API?